geode-core/src/main/java/org/apache/geode/logging/spi/SessionContext.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.logging.spi;
+
+// TODO:KIRK: inline this to use LogConfigSupplier directly instead of SessionContext
```

Comment:
Is this TODO still relevant?  If not, can we remove it?  If so, can we address it in this PR or create a follow up ticket and remove the TODO?

Reply:
It is still relevant but I'll delete it. Thanks!